jsettlers.graphics/src/jsettlers/graphics/map/MapContent.java

```diff
@@ -334,51 +333,44 @@ private void drawMoveToMarker() {
 
 	private float messageAlpha(IMessage m) {
 		int age = (int) m.getAge();
-		return (age < 5000) ?
-			(age < 1500
-				? Math.min(1,  age/1000f) : 1)
+		return age < 1500
+				? Math.min(1,  age/1000f)
 				: Math.max(0,
 					1f - (float)age / IMessage.MESSAGE_TTL);
 	}
 
 	private void drawMessages(GLDrawContext gl) {
 		TextDrawer drawer = textDrawer.getTextDrawer(gl, EFontSize.HEADLINE);
-		// TODO: don't let logic wait until we rendered.
-		synchronized (messenger) {
-			int messageIndex = 0;
-			messenger.doTick();
-			for (IMessage m : messenger.getMessages()) {
-				float x = MESSAGE_OFFSET_X;
-				int y = MESSAGE_OFFSET_Y + messageIndex * MESSAGE_LINEHIEGHT;
-				float a = messageAlpha(m);
-				if (m.getSender() >= 0) {
-					String name = getPlayername(m.getSender()) + ":";
-					Color color = context.getPlayerColor(m.getSender());
-					float width = (float) drawer.getWidth(name);
-					float bright = color.getRed() + color.getGreen() + color.getBlue();
-					if (bright < .9f) {
-						// black
-						drawer.setColor(1, 1, 1, a/2);
-					} else if (bright < 2f) {
-						// bad visibility
-						drawer.setColor(0, 0, 0, a/2);
-					}
-					for (int i=-1; i<3; i++)
-						drawer.drawString(x+i, y-1, name);
-					drawer.setColor(color.getRed(), color.getGreen(),
-							color.getBlue(), a);
-					drawer.drawString(x, y, name);
-					x += width + 10;
+		int messageIndex = 0;
+		messenger.doTick();
+		for (IMessage m : messenger.getMessages()) {
+			float x = MESSAGE_OFFSET_X;
+			int y = MESSAGE_OFFSET_Y + messageIndex * MESSAGE_LINEHIEGHT;
+			float a = messageAlpha(m);
+			if (m.getSender() >= 0) {
+				String name = getPlayername(m.getSender()) + ":";
+				Color color = context.getPlayerColor(m.getSender());
+				float width = (float) drawer.getWidth(name);
+				float bright = color.getRed() + color.getGreen() + color.getBlue();
+				if (bright < .9f) {
+					// black
+					drawer.setColor(1, 1, 1, a/2);
+				} else if (bright < 2f) {
+					// bad visibility
+					drawer.setColor(0, 0, 0, a/2);
 				}
+				for (int i=-1; i<3; i++)
+					drawer.drawString(x+i, y-1, name);
+				drawer.setColor(color.getRed(), color.getGreen(),
+						color.getBlue(), a);
```

Comment:
No, we use 150 characters. Nowadays everybody has a wide screen, so I like to use it. :)